test/JDBC/src/main/resources/config.txt

```diff
@@ -39,6 +39,9 @@ scheduleFile = ./jdbc_schedule
 # WHETHER TEST MODE IS PARALLEL QUERY MODE ON
 isParallelQueryMode = false
 
+# WHETHER DATABASE MIGRATION MODE IS SINGLE-DB
```

Comment:
Should we also maintain a new schedule file for single-db-mode which would be appended over the existing schedule.

Reply:
There is no need to maintain separate schedule file for single-db mode as there are not many test files which are supposed to be run in only single-db mode.